### ISSUE TITLE
Broken link in documentation

### ISSUE THREAD

**Issue by techdude** [MicrosoftDocs/cpp-docs#669]
The link to the "WORDPAD" example is broken.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e7b1a307-d4bf-367b-deae-839d76432099
* Version Independent ID: 16347874-7a1a-159f-6731-b7cd9fa0f6d3
* Content: [Rich Edit Control Examples](https://docs.microsoft.com/en-us/cpp/mfc/rich-edit-control-examples?view=vs-2017#feedback)
* Content Source: [docs/mfc/rich-edit-control-examples.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/mfc/rich-edit-control-examples.md)
* Product: **visual-cpp**
* GitHub Login: @mikeblome
* Microsoft Alias: **mblome**

**Comment by corob-msft** [comment]
@techdude I don't know that I'd call it broken so much as not going directly where you might reasonably expect. The old MFC samples, including [Wordpad](https://github.com/Microsoft/VCSamples/tree/master/VC2010Samples/MFC/ole), are available on GitHub in the [VCSamples](https://github.com/Microsoft/VCSamples) repo linked from the Visual C++ Samples page where the link now takes you.

**Comment by techdude** [comment]
Ah that's where it is. I figured it was broken because I spent about 10 minutes trying to find it and couldn't. Currently the  link goes to a page that has (among options that may make sense) Visual C++ samples, Visual Studio Samples, Windows Desktop code samples, MFC samples. All of these except "Visual C++" go to a search page where searching wordpad doesn't turn up anything, so after clicking through, searching, removing filters and scrolling through results, I figured it was moved.

The link should probably point at the most recent version of the wordpad example, or the version that matches the version closest to the version the person is looking at (in this case it was from the Visual Studio 2017 version of the documentation so it should go to the most recent version [here](https://github.com/Microsoft/VCSamples/tree/da802c2aa92a730b3da33c5957186f128709c398/VC2010Samples/MFC/Visual%20C%2B%2B%202008%20Feature%20Pack/WordPad)). The other option would be to point it at the [vcsamples repo](https://github.com/Microsoft/vcsamples) so at least they can search for it. But right now it points to a page with 10 links, 9 of which direct the user away from what they wanted to find.
